src/Microsoft.ML.CpuMath/CpuMathUtils.netstandard.cs

```diff
@@ -34,6 +40,19 @@ private static bool Compat(AlignedArray a)
             return q;
         }
 
+        /// <summary>
+        /// Compute the product of (flattened because its type is <see cref="AlignedArray"/> instead of a matrix) matrix <paramref name="mat"/>
+        /// and a vector <paramref name="src"/>.
```

Comment:
/// Compute the product of a flattened matrix <paramref name="mat"/> (flattened because its type is <see cref="AlignedArray"/>) 
        /// and a vector <paramref name="src"/>.

Maybe like this it is a bit easier to read.

Reply:
It should be clear shown by intellisense. It's hard to read because git-diff shows raw text.